File: IOS/Quiz_OC/Quiz_OC/ViewController.m
//
//  ViewController.m
//  Quizself.OC
//
//  Created by 田宇 on 2018/4/21.
//  Copyright © 2018年 田宇. All rights reserved.
//

#import "ViewController.h"

@interface ViewController (){
    
}

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self setQuestions: @[@"1+1", @"2+2", @"3+3"]];
    self.answers = @[@"2", @"4", @"6"];
    self.index = 0;
    
    NSLog(@"%@", self.questions);
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)nextQuestion:(id)sender{
    self.index += 1;
    if (self.index == [self.questions count]) {
        self.index = 0;
    }
    NSString *question = [self.questions objectAtIndex:self.index];
    [self.questionLable setText:question];
    [self.answerLable setText:@"???"];
}
- (void)showAnswer:(id)sender{
    NSString *answer = [self.answers objectAtIndex:self.index];
    [self.answerLable setText:answer];
}

@end

File: IOS/Quiz/Quiz/ViewController.swift
//
//  ViewController.swift
//  Quiz
//
//  Created by 田宇 on 2018/4/21.
//  Copyright © 2018年 田宇. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet var questionLable: UILabel!
    @IBOutlet var answerLable: UILabel!
    
    let questions: [String] = ["From what is cognac made?", "What is 7+7?", "What is the captial of Vermont?"]
    let answers: [String] = ["Grapes", "14", "Montpelier"]
    var currentQuestionIndex: Int = 0
    
    @IBAction func showNextQuestion(sender: AnyObject){
        currentQuestionIndex += 1
        if currentQuestionIndex == questions.count {
            currentQuestionIndex = 0
        }
        let question: String = questions[currentQuestionIndex]
        questionLable.text = question
        answerLable.text = "???"
    }
    @IBAction func showAnswer(sender: AnyObject){
        let answer: String = answers[currentQuestionIndex]
        answerLable.text = answer
    }

}

